apps/cesium.omniverse.app.kit

```diff
@@ -17,7 +17,7 @@ app.window.title = "Cesium for Omniverse Test App"
 app.useFabricSceneDelegate = true
 app.usdrt.scene_delegate.enableProxyCubes = false
 app.usdrt.scene_delegate.geometryStreaming.enabled = false
-app.fastShutdown = true
+app.fastShutdown = false
```

Comment:
Can we create a second app file that derives from this one that disables `app.fastShutdown`? Perhaps something called `cesium.omniverse.app.trace.kit`? Without this option debugging some of the Python stuff can be quite a poor experience.

Reply:
Good idea, done!